.github/workflows/test_ssh.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         if: github.repository_owner == 'AndreasAugustin'
         uses: ./ # Uses an action in the root directory
         with:
-          target_gh_token: ${{ secrets.GITHUB_TOKEN }}
+          # target_gh_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
the target token defaults to the GITHUB_TOKEN. so should not be needed.

Reply:
true :+1:

Reply:
@AndreasAugustin what is I specifiy a different token of a different organization? In that case it would not default to the GITHUB_TOKEN. Would that be a problem crossing it out?

Reply:
In that case you put the token into a GitHub secret and use that secret here. But actually the target (currently) is always the repo itself. There is no functionality (yet) to use a different repo as target